### ISSUE TITLE
How to change the

### ISSUE THREAD

**Issue by marsggbo** [melodyguan/enas#79]

**Comment by marsggbo** [comment]
I tried to use ENAS to design architecture on my dataset which is much larger than CIFAR10. If I set the image size to 224*224, then raise ValueError("GraphDef cannot be larger than 2GB."). Is it possible to use placeholder to solve this problem and does anyone succeed? Thanks.

**Comment by Nadav1** [comment]
For classification architecture search the repository [loads all CIFAR10 data to the memory](https://www.tensorflow.org/api_guides/python/reading_data#Preloaded_data).
This is OK for small datasets, but for larger ones its a problem - thus the error you get. 
You need to change the data input pipeline to tf.data to be able to run larger datasets.
See:
https://www.tensorflow.org/guide/datasets

**Comment by marsggbo** [comment]
@Nadav1 Thank you so much. I've solved the problem and put the code in [marsggbo/enas](https://github.com/marsggbo/enas). But now I have another question, i.e. how to use multiple GPUs. Because the code of ENAS is so complex to me, there are many  components, such as controller, child models. So I don't know how to sync the gradients of each GPUs, and I'm not sure whether I need to train controller in parallel. Looking forward to your suggestion. Thanks!

**Comment by ManasiPat** [comment]
@marsggbo I downloaded your code provided above and tried to run it again for cifar10 dataset before running code for my own dataset for which I was getting the ValueError("GraphDef cannot be larger than 2GB.") with original ENAS. For cifar 10 dataset I am getting loss value as nan. 
![screenshot from 2018-12-20 13-31-43](https://user-images.githubusercontent.com/44029189/50271761-b82cf580-045b-11e9-8c0e-431f1f4ce241.png)

**Comment by marsggbo** [comment]
@ManasiPat  Did you modify the code in *sh file. You may need to assign`--output_classes=10` instead of 5.

**Comment by ManasiPat** [comment]
Thanks. I changed that, now I am not getting nan however I am getting my training accuracy to be 0 , 2, or 1
![screenshot from 2018-12-20 14-02-45](https://user-images.githubusercontent.com/44029189/50273179-f1fffb00-045f-11e9-8541-1c04c817639e.png)

**Comment by ManasiPat** [comment]
ok got it. Its number of correct samples out of the batch. Let me run it for a while to see if I get the results as in the original paper. Thanks.

**Comment by ManasiPat** [comment]
@marsggbo  I  am working on age prediction using facial images. Treating it as a classification problem, where I have 101 classes. So, to feed my dataset, I have changed the data_utils file and changed the output_classes=101 and data_path to point at my dataset in the script file. It should work with my data right? Or do I need to make any other changes?

**Comment by tobymu** [comment]
@ManasiPat Hi, I'm also working on age prediction using facial images. Do you success with ENAS?
